### PR TITLE
Add auto-generation used version of media assets (video, favicons, images) script

### PR DESCRIPTION
**Pull request by andriyParashchuk**
We use a lot of optimization tools in order to generate different versions for different devices and get the best performance.

We need to be able to accomplish this without any manual work or at least we should have explicit instruction how to achieve this

- [ ] Optimize images
- [ ] Optimize videos
- [ ] Generate favicons

there is jekyl or gulp plugins which add this in build stage

### CODE REVIEW

**Comment by pftg** [comment]
and setup script to auto do converting, I assume that there are some jekyll plugins

**Comment by andriyParashchuk** [comment]
@pftg @obliviusm @sweatC pls check

File: README.md [hunk 1]
```diff
@@ -39,3 +39,27 @@
         More detailed explanatory ...
 
 1. Convert an existing issue into a pull request: `hub pull-request -i [issue-number]`
+
+## How use auto-generation of media assets:
+
+1. Optimize images: https://tinypng.com/
+    - Upload your original image (.png, jpg)
+    - Click download
+    * It is important that the picture is sufficiently optimized and in good quality
```
Comment (pftg) [inline README.md]: `*` => `-`

File: README.md [hunk 2]
```diff
@@ -39,3 +39,27 @@
...
+    And video of two formats: mp4, m4v
+    - Upload your original video
+    - Select the desired resolution and format
+    - click convert and after download
+3. Generate favicons: https://www.favicon-generator.org/
+    We use icons for the site size: 16x16, 32x32x 144x144
+    - Upload your original image
+    - Click Create Favicon after download
+    - Connect icons in the tag head
+    * It is important that the original image is exactly equal proportions of the sides
```
Comment (pftg) [inline README.md]: do we need to have @x2 versions?

File: README.md (hunk 1 above)
Comment (pftg) [reply README.md]: what about @x2 versions?

File: README.md [hunk 3]
```diff
@@ -39,3 +39,27 @@
...
+4. Generate fonts: https://fontie.pixelsvsbytes.com/webfont-generator
+    - Click the Format tab, and select: TrueType Font, Web Open Font Format, Web Open Font Format 2
+    - Click Generate & download
+    - Put the fonts in the folder _assets/fonts
```
Comment (pftg) [inline README.md]: where is original files stored?

**Review by pftg — COMMENTED** [review]
Where is original files hosted?

File: README.md (hunk 3 above)
Comment (andriyParashchuk) [reply README.md]: _assets/fonts/.ttf

File: README.md (hunk 2 above)
Comment (andriyParashchuk) [reply README.md]: we do not have images quality x2

File: README.md (hunk 1 above)
Comment (andriyParashchuk) [reply README.md]: accepted

**Comment by andriyParashchuk** [comment]
@pftg pls check

**Review by pftg — COMMENTED** [review]
LGTM

**Pull request merged**